itemizedReceipt.js

```diff
@@ -6,13 +6,67 @@
 
 
 // Check
-logReceipt(
-  { descr: 'Burrito', price: 5.99 },
-  { descr: 'Chips & Salsa', price: 2.99 },
-  { descr: 'Sprite', price: 1.99 }
-);
+let logReceipt =[
+  { descr: 'Burrito', 
+    price: 5.99,
+    type: 'Food'
+  },
+  { descr: 'Chips & Salsa', 
+    price: 2.99,
+    type: 'Food'
+  },
+  { descr: 'Sprite', 
+    price: 1.99,
+    type:'Drink',
+    oz: 12,
+    sugar: true
+  }
+]
+  ;
 // should log something like:
 // Burrito - $5.99
 // Chips & Salsa - $2.99
 // Sprite - $1.99
 // Total - $10.97
+
+// source: https://dor.wa.gov/get-form-or-publication/publications-subject/tax-topics/seattle-s-sweetened-beverage-tax
+const seattleSugarTaxRate = .0175
+
+//source: http://www.salestaxstates.com/sales-tax-calculator-washington-seattle-98101
+const washingtonSalesTax = .065
+const seattleSaleTax = .0375
+const wholeSalesTax = washingtonSalesTax + seattleSaleTax
+
+function Receipt(){
+  let sugarTaxTotal = 0;
+  let totalBeforeTax = 0;
+  let totalSalesTax = 0;
+  totalWithTax = 0;
+  console.log(`
+        THANK YOU FOR SHOPPING AT CARLITO's 
+  -----------------------------------------------------------------
+  ITEM - PRICE`)
+  for(let i =0; i<logReceipt.length; i++){
+    if(logReceipt[i].type == 'Drink'){
+      if(logReceipt[i].sugar == true){
+        sugarTaxTotal += logReceipt[i].oz * seattleSugarTaxRate
+      }
+    }
+    totalBeforeTax +=logReceipt[i].price
+    totalSalesTax +=logReceipt[i].price*wholeSalesTax
+    console.log(`
+  ${logReceipt[i].descr} - $${logReceipt[i].price}`)
+  }
+  totalWithTax += totalBeforeTax + totalSalesTax + sugarTaxTotal
+  console.log(`
+  -----------------------------------------------------------------
+  Amount:                      $${totalBeforeTax}
+  Sales Tax:                   $${totalSalesTax.toFixed(2)}
+  Seattle's Sugar Tax:         $${sugarTaxTotal.toFixed(2)}
+  -----------------------------------------------------------------
+  Total:                       $${totalWithTax.toFixed(2)} 
+  -----------------------------------------------------------------
+                  THANK YOU!
+  `)
+}
+Receipt(logReceipt)
```

Comment:
The formatting is really nice, but unfortunately this implementation negates the idea of having a `logReceipt()` function. Keep in mind with these starter files that you don't want to change the part that is already there . . . The setup _wants_ to run a function named `logReceipt()` that you are tasked with writing, and then give it those provided values. 

So a couple lessons here . . . `logReceipt()` is meant to be a reusable piece of code that can take any number of given parameters. This is the core of writing functions, being able to use that bit of code over and over again with variable inputs. One key tool that needs to be used is a spread operator, so that it will work if given 1,2,3, or 1000 menu items.

What's happened here instead is making `logReceipt()` just a static array, and not a function at all, then getting the desired output with a static, single-purpose function `Receipt()` that can only ever do this one job.

Try again, simplify things, and let the boilerplate run `logReceipt()` as a function that takes the three objects.